Examples/Algorithms/TrackFinding/src/TrackFindingAlgorithm.cpp

```diff
@@ -536,10 +536,13 @@ ProcessCode TrackFindingAlgorithm::execute(const AlgorithmContext& ctx) const {
         }
 
         if (firstMeasurementOpt.has_value()) {
-          auto& firstMeasurement = firstMeasurementOpt.value();
+          TrackContainer::TrackStateProxy firstMeasurement{
+              firstMeasurementOpt.value()};
+          TrackContainer::ConstTrackStateProxy firstMeasurementConst{
+              firstMeasurement};
```

Comment:
```suggestion
          TrackContainer::ConstTrackStateProxy firstMeasurementConst{
              firstMeasurementOpt.value()};
```
does not work because of an intermediate conversion step?